src/Raven.Studio/typescript/commands/database/query/queryCommand.ts

```diff
@@ -124,17 +124,34 @@ f();
         return JSON.stringify(payload);
     }
     
-    getUrl() {
+    getUrl(GETRequest: boolean = false) {
```

Comment:
paramaters names in ts starts with lowercase, maybe let's do:
getUrl(method: "POST" | "GET" = "POST") ?

Reply:
then we avoid having: 
queryCmd.getUrl(true)
since no one knows what this true means.

Reply:
done